keyboards/ropro/info.json

```diff
@@ -0,0 +1,13 @@
+{
+  "keyboard_name": "ropro",
+  "url": "https://github.com/TheRoyalSweatshirt/The_RoPro",
+  "maintainer": "[Garret G.](https://github.com/TheRoyalSweatshirt)",
+  "width": 13,
+  "height": 6,
+  "layouts": {
+    "LAYOUT_ortho_6x13": {
+      "key_count": 76,
+      "layout": [{"label":"CAPS", "x":0, "y":2}, {"label":"PgUp", "x":0, "y":3}, {"label":"Home", "x":0, "y":4}, {"label":"PgDn", "x":0, "y":5}, {"label":"F1", "x":1, "y":0}, {"label":"Esc", "x":1, "y":1}, {"label":"Tab", "x":1, "y":2}, {"label":"Ctrl", "x":1, "y":3}, {"label":"Shift", "x":1, "y":4}, {"label":"Del", "x":1, "y":5}, {"label":"F2", "x":2, "y":0}, {"label":"1", "x":2, "y":1}, {"label":"Q", "x":2, "y":2}, {"label":"A", "x":2, "y":3}, {"label":"Z", "x":2, "y":4}, {"label":"Ctrl", "x":2, "y":5}, {"label":"F3", "x":3, "y":0}, {"label":"2", "x":3, "y":1}, {"label":"W", "x":3, "y":2}, {"label":"S", "x":3, "y":3}, {"label":"X", "x":3, "y":4}, {"label":"GUI", "x":3, "y":5}, {"label":"F4", "x":4, "y":0}, {"label":"3", "x":4, "y":1}, {"label":"E", "x":4, "y":2}, {"label":"D", "x":4, "y":3}, {"label":"C", "x":4, "y":4}, {"label":"Alt", "x":4, "y":5}, {"label":"F5", "x":5, "y":0}, {"label":"4", "x":5, "y":1}, {"label":"R", "x":5, "y":2}, {"label":"F", "x":5, "y":3}, {"label":"V", "x":5, "y":4}, {"label":"Lower", "x":5, "y":5}, {"label":"F6", "x":6, "y":0}, {"label":"5", "x":6, "y":1}, {"label":"T", "x":6, "y":2}, {"label":"G", "x":6, "y":3}, {"label":"B", "x":6, "y":4}, {"x":6, "y":5}, {"label":"F7", "x":7, "y":0}, {"label":"6;", "x":7, "y":1}, {"label":"Y", "x":7, "y":2}, {"label":"H;", "x":7, "y":3}, {"label":"N", "x":7, "y":4}, {"x":7, "y":5},{"label":"F8", "x":8, "y":0}, {"label":"7", "x":8, "y":1}, {"label":"U", "x":8, "y":2}, {"label":"J", "x":8, "y":3}, {"label":"M", "x":8, "y":4}, {"label":"End", "x":8, "y":5}, {"label":"F9", "x":9, "y":0, {"label":"8", "x":9, "y":1}}, {"label":"I", "x":9, "y":2}, {"label":"K", "x":9, "y":3}, {"label":",", "x":9, "y":4}, {"label":"Left", "x":9, "y":5}, {"label":"F10", "x":10, "y":0}, {"label":"9", "x":10, "y":1}, {"label":"O", "x":10, "y":2}, {"label":"L", "x":10, "y":3}, {"label":".", "x":10, "y":4}, {"label":"Down", "x":10, "y":5}, {"label":"F11", "x":11, "y":0}, {"label":"0", "x":11, "y":1}, {"label":"P", "x":11, "y":2}, {"label":";", "x":11, "y":3}, {"label":"/", "x":12, "y":4}, {"label":"Up", "x":11, "y":5}, {"label":"F12", "x":12, "y":0}, {"label":"-", "x":12, "y":1}, {"label":"Bksp", "x":12, "y":2}, {"label":"'", "x":12, "y":3}, {"label":"Enter", "x":12, "y":4}, {"label":"Right", "x":12, "y":5}]
```

Comment:
Your JSON objects are out of sequence here. The keys need to be in the same order that `keymap.c` uses.

```suggestion
      "layout": [{"label":"F1", "x":1, "y":0}, {"label":"F2", "x":2, "y":0}, {"label":"F3", "x":3, "y":0}, {"label":"F4", "x":4, "y":0}, {"label":"F5", "x":5, "y":0}, {"label":"F6", "x":6, "y":0}, {"label":"F7", "x":7, "y":0}, {"label":"F8", "x":8, "y":0}, {"label":"F9", "x":9, "y":0}, {"label":"F10", "x":10, "y":0}, {"label":"F11", "x":11, "y":0}, {"label":"F12", "x":12, "y":0}, {"label":"Esc", "x":1, "y":1}, {"label":"1", "x":2, "y":1}, {"label":"2", "x":3, "y":1}, {"label":"3", "x":4, "y":1}, {"label":"4", "x":5, "y":1}, {"label":"5", "x":6, "y":1}, {"label":"6;", "x":7, "y":1}, {"label":"7", "x":8, "y":1}, {"label":"8", "x":9, "y":1}, {"label":"9", "x":10, "y":1}, {"label":"0", "x":11, "y":1}, {"label":"-", "x":12, "y":1}, {"label":"CAPS", "x":0, "y":2}, {"label":"Tab", "x":1, "y":2}, {"label":"Q", "x":2, "y":2}, {"label":"W", "x":3, "y":2}, {"label":"E", "x":4, "y":2}, {"label":"R", "x":5, "y":2}, {"label":"T", "x":6, "y":2}, {"label":"Y", "x":7, "y":2}, {"label":"U", "x":8, "y":2}, {"label":"I", "x":9, "y":2}, {"label":"O", "x":10, "y":2}, {"label":"P", "x":11, "y":2}, {"label":"Bksp", "x":12, "y":2}, {"label":"PgUp", "x":0, "y":3}, {"label":"Ctrl", "x":1, "y":3}, {"label":"A", "x":2, "y":3}, {"label":"S", "x":3, "y":3}, {"label":"D", "x":4, "y":3}, {"label":"F", "x":5, "y":3}, {"label":"G", "x":6, "y":3}, {"label":"H;", "x":7, "y":3}, {"label":"J", "x":8, "y":3}, {"label":"K", "x":9, "y":3}, {"label":"L", "x":10, "y":3}, {"label":";", "x":11, "y":3}, {"label":"'", "x":12, "y":3}, {"label":"Home", "x":0, "y":4}, {"label":"Shift", "x":1, "y":4}, {"label":"Z", "x":2, "y":4}, {"label":"X", "x":3, "y":4}, {"label":"C", "x":4, "y":4}, {"label":"V", "x":5, "y":4}, {"label":"B", "x":6, "y":4}, {"label":"N", "x":7, "y":4}, {"label":"M", "x":8, "y":4}, {"label":",", "x":9, "y":4}, {"label":".", "x":10, "y":4}, {"label":"/", "x":11, "y":4}, {"label":"Enter", "x":12, "y":4}, {"label":"PgDn", "x":0, "y":5}, {"label":"Del", "x":1, "y":5}, {"label":"Ctrl", "x":2, "y":5}, {"label":"GUI", "x":3, "y":5}, {"label":"Alt", "x":4, "y":5}, {"label":"Lower", "x":5, "y":5}, {"x":6, "y":5}, {"x":7, "y":5}, {"label":"End", "x":8, "y":5}, {"label":"Left", "x":9, "y":5}, {"label":"Down", "x":10, "y":5}, {"label":"Up", "x":11, "y":5}, {"label":"Right", "x":12, "y":5}]
```